Userland/DevTools/HackStudio/CMakeLists.txt

```
@@ -11,6 +13,9 @@ set(SOURCES
     Debugger/DisassemblyWidget.cpp
     Debugger/RegistersModel.cpp
     Debugger/VariablesModel.cpp
+    Dialogs/NewProjectDialog.cpp
+    Dialogs/NewProjectDialog.gml
```

This should be

```suggestion
    Dialogs/NewProjectDialogGML.h
```

I think...

e.g. in Browser:

https://github.com/SerenityOS/serenity/blob/dc28c07fa526841e05e16161c74a6c23984f1dd5/Userland/Applications/Browser/CMakeLists.txt#L1-L16